telegram-bot-api/HttpStatConnection.cpp

```diff
@@ -23,7 +23,9 @@ void HttpStatConnection::handle(td::unique_ptr<td::HttpQuery> http_query,
   init_promise_future(&promise, &future);
   future.set_event(td::EventCreator::yield(actor_id()));
   LOG(DEBUG) << "SEND";
-  send_closure(client_manager_, &ClientManager::get_stats, std::move(promise), http_query->get_args());
+  td::Parser url_path_parser(http_query->url_path_);
+  bool as_json = url_path_parser.try_skip("/json");
```

Comment:
should become an attribute

Reply:
this could lead to a race condition, see the discussion [here](https://t.me/TDLightChat/1455)

Reply:
Attribute?

Reply:
Please explain what you mean?